exonum/src/node/consensus.rs

```diff
@@ -689,6 +691,7 @@ impl NodeHandler {
 
     /// Handles propose timeout. Node sends `Propose` and `Prevote` if it is a leader as result.
```

Comment:
Shouldn't we mention that this function allows expedited propose now?

Reply:
Mmm... I'm not sure this will be interesting for its users. Resetting a flag is just an implementation detail.

Maybe we should add comments to `handle_tx*` functions which tell that as a side effect of handling transactions we now may schedule a Propose timeout. But I'm not sure whether that is interesting enough as well.

Reply:
Should we mention this in changelog or this is minor enough?

Reply:
I'd prefer to mention this both in `changelog` and documentation. However, it's just my opinion. If you doubt, you can discuss it with the rest of the team :)

Reply:
@DarkEld3r wdyt?

Reply:
This comments are mostly for us, consensus (`node` module) should ideally be internal. This way the comments should be updated when it makes sense (I'm against too verbose comments).